hack/build-rpms.sh

```diff
@@ -58,12 +58,14 @@ if [[ -n "${dirty}" && "${OS_GIT_TREE_STATE}" == "dirty" ]]; then
 		--define "_sourcedir ${rpm_tmp_dir}/SOURCES" \
 		--define "_builddir ${rpm_tmp_dir}/BUILD" \
 		--define "skip_prep 1" \
-		--define "skip_dist 1" \
+		--define "skip_dist ${SKIP_DIST:-1}" \
+		--define "ose_build ${OSE_BUILD:-1}" \
```

Comment:
ose or ocp build?

Reply:
`ocp_build` if you have too.

Reply:
won't be necessary if we can use `openshift` name schema